src/store/modules/loadingStatus.ts

```diff
@@ -0,0 +1,30 @@
+import { Action, Module, Mutation, VuexModule } from 'vuex-module-decorators'
+
```

Comment:
I think better module name will be loader

Reply:
renamed :)